book/02-git-basics/sections/viewing-history.asc

```diff
@@ -132,6 +132,10 @@ Sie enthält auch eine Zusammenfassung am Ende des Berichts.
 
 Eine weitere wirklich nützliche Option ist `--pretty`.
 Diese Option ändert das Format der Log-Ausgabe in ein anderes als das Standard-Format.
+// Hier ist das englische Original wieder suboptimal (wie ich finde).
```

Comment:
Super Entdeckung! 
Diesen Bereich habe ich, in völliger Ahnungslosigkeit, so ziemlich zu Beginn der meiner Übersetzungen bearbeitet.
Wenn hier "Argument" statt "Option" hingehört, dann ändere das.

Dazu heißt es in der Git Manpage `git-log(1)` nur:
```
--pretty[=<format>]::
--format=<format>::

	Pretty-print the contents of the commit logs in a given format,
	where '<format>' can be one of 'oneline', 'short', 'medium',
	'full', 'fuller', 'reference', 'email', 'raw', 'format:<string>'
	and 'tformat:<string>'.  When '<format>' is none of the above,
	and has '%placeholder' in it, it acts as if
	'--pretty=tformat:<format>' were given.
```

Du solltest dir wirklich überlegen, ob du dazu nicht doch ein Issue im englischen Repo startest.
Ich könnte das nicht richtig formulieren.

Reply:
Ich (vermutlich) auch nicht. Hoffen wir auf @cbachhuber, der sich erst einmal vorwagen kann. Es es dann doch Diskussionsbedarf geben sollte, kann ich mich dann ja einschalten.

Reply:
Das ist tatsächlich ein sehr guter Punkt. Wenn man es genau nimmt, sind `--stat`, `--pretty` etc. Argumente, und nicht Optionen für `git log`. Allerdings verwendet selbst [argparse von Python](https://docs.python.org/3/library/argparse.html) (argument parser, hier ist das 'Argument' sogar im Namen) oft `argument` und `option` synonym.

Ich würde vorschlagen, dass wir hier von Optionen/Argumenten (`--pretty`) und Optionswerten (`oneline`) schreiben. Sowas könnte ich auch im englischen Repo vorschlagen. Was haltet ihr davon?

Reply:
einverstanden!

Reply:
Die letztendlichen Namen sind mir fast egal. Nur zweimal von Optionen zu sprechen ist halt ziemlich suboptimal. Gerne dürft/solltet ihr das auch im englischen Repo vorschlagen.

Reply:
Auch hier schlage ich vor, dies in einem separaten Commit zu machen. Dann ist auch dies klar. Einverstanden?

Reply:
Nun ganz egal ist das nicht – es sollten schon konsequent die gleichen Bezeichnungen verwendet werden.
Besonders auch im Blick auf parallele Doku's, wie die Manpages. Zumindest einer der Maintainer dort, schaut manchmal hier nach wie übersetzt wurde.
@cbachhuber könntest du bitte im englischen Repo nachfragen?

Reply:
OK

Reply:
Mache ich! PR/Issue kommt heute noch

Reply:
Offen, eure Meinung dazu ist gern gesehen: https://github.com/progit/progit2/pull/1457

Reply:
@cbachhuber  Super, könntest du bitte auch die Übersetzung hier übernehmen?

Reply:
Gerne, mache ich. @Mo-Gul hat dafür #120 aufgemacht.